odml/tools/rdf_converter.py

```diff
@@ -49,13 +49,12 @@ def __init__(self, odml_documents):
         subclass_path = os.path.join(dirname(dirname(dirname(abspath(__file__)))),
                          'doc', 'section_subclasses.yaml')
 
-        if os.path.isfile(subclass_path):
-            with open(subclass_path, "r") as f:
-                try:
-                    self.section_subclasses = yaml.load(f)
-                except yaml.parser.ParserError as err:
-                    print(err)
-                    return
+        with open(subclass_path, "r") as f:
```

Comment:
This approach might work for a proper PyPI release, but will not work for a local install via `pip install .` or `python setup.py install`